packages/editor/src/client/panel/publishArticlePanel.tsx

```diff
@@ -45,29 +45,56 @@ export function PublishArticlePanel({
         )}
         <Form fluid={true}>
           <FormGroup>
-            <ControlLabel>{t('articleEditor.panels.publishDate')}</ControlLabel>
+            <ControlLabel>{t('articleEditor.panels.publishTime')}</ControlLabel>
```

Comment:
Please set the date field in the first position. Users will more likely change the date then the time